async-http-client/src/main/scala/org/http4s/client/asynchttpclient/AsyncHttpClient.scala

```diff
@@ -146,6 +139,7 @@ object AsyncHttpClient {
       }
   }
 
+  import cats.syntax._
```

Comment:
is this used? There's a `cats.implicits._` import on top of this file

Reply:
I think it might be to dance around the unused import warning, but we have a different idiom.